.ci

```diff
@@ -32,7 +32,6 @@ pipeline {
     LANGUAGE = "en_US:en"
     LC_ALL = "en_US.UTF-8"
     PRODUCTION_DEPLOY="false"
-    SLACK_CHANNEL = "#staging-releases"
```

Comment:
Sorry I made a wrong comment earlier. You do need `SLACK_CHANNEL` if you don't give `SLACK_SUCCESS_CHANNEL` and/or `SLACK_FAILURE_CHANNEL`. And that's why this failed:

https://test.apartmentlist.io/blue/organizations/jenkins/apartmentlist%2Fbulk-processor/detail/PR-34-head/9/pipeline